components/SiteNavigation.vue

```diff
@@ -11,15 +11,15 @@
         :default-open="currentPath.startsWith(item.path)"
       >
         <DisclosureButton
-          class="group mt-2 w-full flex items-center pl-2 pr-1 py-2 text-left text-content-primary text-base leading-6 font-medium rounded-md hover:bg-gray-200 focus:outline-none focus:ring-2 focus:ring-inset ring-focus-ring"
+          class="group mt-2 w-full flex items-center pl-2 pr-1 py-2 space-x-2 text-left rounded-md hover:bg-gray-200 focus:outline-none focus:ring-2 focus:ring-inset ring-focus-ring"
           :class="currentPath.startsWith(item.path) ? 'bg-gray-100' : ''"
         >
           <component
```

Comment:
The code from the icon was already in there from when the side panel was originally made 👀